dist/delorean.js

```diff
@@ -260,21 +267,21 @@
         this.setValue(arg1, value);
       }
       this.recalculate(changedProps);
-      return this.store[arg1];
+      return this.state[arg1];
     };
 
     // `set` method updates the data defined at the `scheme` of the store.
     Store.prototype.setValue = function (key, value) {
-      var scheme = this.store.scheme, definition;
-      if (scheme && this.store.scheme[key]) {
+      var scheme = this.scheme, definition;
+      if (scheme && this.scheme[key]) {
         definition = scheme[key];
 
         // This will allow you to directly set falsy values before falling back to the definition default
-        this.store[key] = (typeof value !== 'undefined') ? value : definition.default;
+        this.state[key] = (typeof value !== 'undefined') ? value : definition.default;
 
         if (typeof definition.calculate === 'function') {
-          this.store[__generateOriginalName(key)] = value;
-          this.store[key] = definition.calculate.call(this.store, value);
+          this.state[__generateOriginalName(key)] = value;
+          this.state[key] = definition.calculate.call(this.state, value);
```

Comment:
I think this should be executed in the `this` context, was there any reason you decided to use `this.state`. With this context `this` refers to `state` within `calculate` methods, I think it's more useful to be up a level and have full access to your store with `calculate methods.